bootstrap-salt.sh

```diff
@@ -4461,7 +4461,11 @@ install_amazon_linux_ami_deps() {
     # Shim to figure out if we're using old (rhel) or new (aws) rpms.
     _USEAWS=$BS_FALSE
 
-    repo_rev="$(echo "${GIT_REV:-$STABLE_REV}"  | sed 's|.*\/||g')"
+    if [ "$ITYPE" = "stable" ]; then
+        repo_rev="$(echo "${STABLE_REV}"  | sed 's|.*\/||g')"
+    else
+        repo_rev="latest"
+    fi
```

Comment:
I'm not sure why we need this `repo_rev` variable at all, since `STABLE_REV` will always contain `latest` during `git` bootstrap mode. https://github.com/saltstack/salt-bootstrap/blob/develop/bootstrap-salt.sh#L533